src/pages/404.astro

```diff
@@ -0,0 +1,47 @@
+---
+import Link from "../components/Link.astro";
+import PageIntro from "../components/PageIntro.astro";
+import Layout from "../layouts/Layout.astro";
+---
+
+<Layout title="404 Not Found • Catppuccin" description="Oops! The page you're looking for could not be found.">
+  <div class="wrapper">
+    <div class="page-content">
+      <PageIntro title="404 Not Found">
+        <p>Oops! The page you're looking for could not be found.</p>
+        <p>Please check the URL or go back to the <Link href="/" external>homepage</Link>.</p>
+      </PageIntro>
+    </div>
+  </div>
+</Layout>
+
+<style lang="scss" is:global>
+  @use "../styles/utils";
+
+  :root {
+    --accent-color: var(--peach);
+  }
+
+  .wrapper {
+    display: flex;
+    flex-direction: column;
+    min-height: 100vh;
+  }
+
+  .page-content {
+    flex: 1;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+  }
+
+  h1,
+  p {
+    text-align: center;
+  }
+
+  footer {
+    margin-top: auto;
+  }
```

Comment:
I believe this CSS incorrectly shifts the footer content:

![image](https://github.com/catppuccin/website/assets/58985301/6f12afab-0a6e-4adc-9bcf-474e664f6e59)

I'd expect this to stay the same as the other pages:

![image](https://github.com/catppuccin/website/assets/58985301/2b45da16-4f20-499d-a339-06532508c166)